templates/gce.tf.erb

```diff
@@ -1,12 +1,6 @@
-provider "google" {
-  region      = "<%= ENV['GCE_REGION'] %>"
-  project     = "govuk-publishing-dns"
-  credentials = "${file("<%= ENV['GCE_CREDENTIALS'] %>")}"
-}
-
 <% records['records'].each_with_index do | record, index | %>
 resource "google_dns_record_set" "<%= record['subdomain'].gsub('.', '_').gsub('@', 'AT') %>_<%= index %>" {
```

Comment:
@deanwilson & @SamLR 

I think these points should be recorded elsewhere??  Perhaps in a Trello ticket.  There valid but off scope here and likely to be forgotten once this PR goes away.